presto-main/src/main/java/com/facebook/presto/operator/aggregation/fixed_histogram/FixedHistogramUtils.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.aggregation.fixedhistogram;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+class FixedHistogramUtils
+{
+    private FixedHistogramUtils() {}
+
+    public static void verifyParams(int bucketCount, double min, double max)
```

Comment:
spell out function names: `verifyHistogramParameters `

Reply:
OK to change to `verifyParameters`? `Histogram` is already in the class name. Also, for consistency, why is this method different in name than the other ones?